todo_app/data/trello_items.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import os
+from todo_app.data.Item import Item
+
+
+def build_parameters(extraparams={}):
+    query = {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN')}
+    query.update(extraparams)
+    return query
+
+
+def build_url(extrapart=""):
+    url = "https://api.trello.com/1/" + extrapart
+    return url
+
+
+def get_lists():
+    url = build_url("boards/" + os.getenv('BOARD_ID') + "/lists")
```

Comment:
You can use string interpolation here to make it a bit cleaner:

```suggestion
    board_id = os.getenv('BOARD_ID')
    url = build_url(f"boards/{board_id}/lists")
```